content/en/agent/basic_agent_usage/_index.md

```diff
@@ -256,13 +256,13 @@ Enabling JMX Checks forces the Agent to use more memory depending on the number
 {{< tabs >}}
 {{% tab "Agent v6 & v7" %}}
 
-* Agent Test version: 6.7.0
-* CPU: ~ 0.12% of the CPU used on average
-* Memory: ~ 60MB of RAM used (RSS memory)
-* Network bandwidth: ~ 86 B/s ▼ | 260 B/s ▲
+* Agent Test version: 7.34.0
+* CPU: ~ 0.08% of the CPU used on average
+* Memory: ~ 130MB of RAM used (RSS memory)
```

Comment:
is this for the core agent process only?

Reply:
Yes, got these metrics using the process-agent but looked only at the core agent process.